src/assets/user-info-settings/index.css

```
@@ -0,0 +1,298 @@
+	color: #000;
+}
+
+div.group > select {
+	content: ' ';
+	background-color: #FFF;
+	background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='32' height='32' fill='%23673DB6' viewBox='0 0 256 256'%3E%3Cpath d='M213.66,101.66l-80,80a8,8,0,0,1-11.32,0l-80-80A8,8,0,0,1,53.66,90.34L128,164.69l74.34-74.35a8,8,0,0,1,11.32,11.32Z'%3E%3C/path%3E%3C/svg%3E");
+	background-size: 40px;
+	background-position: 440px center;
+	background-repeat: no-repeat;
+	color: #9D6FF3;
+	cursor: pointer;
+}
+
+
+div.group > span {
+	color: #000;
+  }
+  div.group > svg {
+	fill: #9D6FF3;
+	border: none;
+	height: 40px;
+	width: 40px;
+	box-sizing: border-box;
+	user-select: none;
+	padding: 0px;
+	margin-bottom: -10px;
+	margin-left: 5px;
+  }
+
+.account-info {
+	float: left;
+	width: 200px;
+	box-sizing: content-box;
+	-webkit-box-sizing: content-box;
+	-webkit-box-align: center;
+	-webkit-box-pack: center;
+}
+
+.account-info > img {
+	display: inline-block;
+	width: 128px;
+	height: 128px;
+	overflow: hidden;
+	border-radius: 100%;
+	background: #E4DBF2;
+	margin: 30px 30px 0 35px;
+	box-shadow: 0-1px 2px rgba(0, 0, 0, 0.4);
+}
+
+.account-info > .content {
+	text-align: center;
+	color: #673DB6;
+	margin: 0;
+}
+
+.account-info > h3.content {
+	font-weight: normal;
+	color: #9D6FF3;
+	font-size: 20px;
+}
+
+.account-info .access-level-banned {
+	background: rgba(255, 63, 0, 0.3);
+	color: #FF3F00;
+	border-color: #FF3F00;
+}
+
+.account-info .tier-level-1 {
+	background: #934D4D;
+	color: #FF8484;
+	border-color: #FF8484;
+}
+
+.account-info .tier-level-2 {
+	background: #316C59;
+	color: #59C9A5;
+	border-color: #59C9A5;
+}
+
+.account-info .tier-level-3 {
+	background: #6B5E84;
+	color: #CAB1FB;
+	border-color: #CAB1FB;
+}
+
+.account-info .access-level-1 {
+	background: #3B918C;
+	color: #64F7EF;
+	border-color: #64F7EF;
+}
+
+.account-info .access-level-2 {
+	background: #917235;
+	color: #FFC759;
+	border-color: #FFC759;
+}
+
+.account-info .access-level-3 {
+	background: #3A973C;
+	color: #5AFF15;
+	border-color: #5AFF15;
+}
+
+.account-info .tier-name {
+	margin: 12px auto;
+	line-height: 1.2em;
+	border-radius: 1.2em;
+	border-width: 2px;
+	border-style: solid;
+	padding: 4px 16px;
+	width: min-content;
+	display: inline-block;
+}
+
+div.group > p.content > button {
+	min-width: 100px;
+	min-height: 20px;
+	display: block;
+	width: 100px;
+	float: right;
+}
+
+div.radio {
+	display: inline-block;
+}
+
+.server-selection input {
+	opacity: 0;
+	position: absolute;
+	width: 200px;
+	height: 70px;
+	cursor: pointer;
+}
+
+div.radio label {
+	display: inline-block;
+	color: #6A6C75;
+	min-width: 200px;
+	box-sizing: content-box;
+	-webkit-box-sizing: content-box;
+	-webkit-box-align: center;
+	-webkit-box-pack: center;
+	text-align: center;
+	background: -webkit-gradient(linear, left top, left bottom, from(#FFF), color-stop(0.5, #FFF), color-stop(0.8, #F6F6F6), color-stop(0.95, #F5F5F5), to(#BBB))0 0;
```

Nitpick. Missing space

```suggestion
	background: -webkit-gradient(linear, left top, left bottom, from(#FFF), color-stop(0.5, #FFF), color-stop(0.8, #F6F6F6), color-stop(0.95, #F5F5F5), to(#BBB)) 0 0;
```